shared/src/test/scala/FetchTests.scala

```diff
@@ -98,916 +98,3 @@ object TestHelper {
   def totalBatches(rs: Seq[Round]): Int =
     rs.map((round: Round) => requestBatches(round.request)).toList.sum
 }
-
-class FetchSyntaxTests extends AsyncFreeSpec with Matchers {
-  import fetch.syntax._
-  import TestHelper._
-
-  val ME = implicitly[FetchMonadError[Future]]
-
-  implicit override def executionContext = ExecutionContext.Implicits.global
-
-  "Cartesian syntax is implicitly concurrent" in {
-    import cats.syntax.cartesian._
-
-    val fetch: Fetch[(Int, List[Int])] = (one(1) |@| many(3)).tupled
-
-    val fut = Fetch.runEnv[Future](fetch)
-
-    fut.map(env => {
-      env.rounds.size shouldEqual 1
-    })
-  }
-
-  "Apply syntax is implicitly concurrent" in {
-    import cats.syntax.apply._
-
-    val fetch: Fetch[Int] = Fetch.pure((x: Int, y: Int) => x + y).ap2(one(1), one(2))
-
-    val fut = Fetch.runEnv[Future](fetch)
-
-    fut.map(env => {
-      val rounds = env.rounds
-
-      rounds.size shouldEqual 1
-      totalBatches(rounds) shouldEqual 1
-      totalFetched(rounds) shouldEqual 2
-    })
-  }
-
-  "`fetch` syntax allows lifting of any value to the context of a fetch" in {
-    Fetch.pure(42) shouldEqual 42.fetch
-  }
-
-  "`fetch` syntax allows lifting of any `Throwable` as a failure on a fetch" in {
-    case object Ex extends RuntimeException
-
-    val fut1 = Fetch.run[Future](Fetch.error(Ex): Fetch[Int])
-    val fut2 = Fetch.run[Future](Ex.fetch: Fetch[Int])
-
-    val e1 = ME.handleErrorWith(fut1)(err => Future.successful(42))
-    val e2 = ME.handleErrorWith(fut2)(err => Future.successful(42))
-
-    ME.map2(e1, e2)(_ shouldEqual _)
-  }
-
-  "`join` syntax is equivalent to `Fetch#join`" in {
-    val join1 = Fetch.join(one(1), many(3))
-    val join2 = one(1).join(many(3))
-
-    ME.map2(Fetch.run[Future](join1), Fetch.run[Future](join2))(_ shouldEqual _)
-  }
-
-  "`runF` syntax is equivalent to `Fetch#runFetch`" in {
-
-    val rf1 = Fetch.runFetch[Future](1.fetch)
-    val rf2 = 1.fetch.runF[Future]
-
-    ME.map2(rf1, rf2)(_ shouldEqual _)
-  }
-
-  "`runE` syntax is equivalent to `Fetch#runEnv`" in {
-
-    val rf1 = Fetch.runEnv[Future](1.fetch)
-    val rf2 = 1.fetch.runE[Future]
-
-    ME.map2(rf1, rf2)(_ shouldEqual _)
-  }
-
-  "`runA` syntax is equivalent to `Fetch#run`" in {
-
-    val rf1 = Fetch.run[Future](1.fetch)
-    val rf2 = 1.fetch.runA[Future]
-
-    ME.map2(rf1, rf2)(_ shouldEqual _)
-  }
-}
-
-class FetchTests extends AsyncFreeSpec with Matchers {
```

Comment:
Have we dropped these ?
Seems we are removing around 550 lines in this PR, which is more or less the same as these tests ?